frontend/src/app/containers/ExperimentPage.js

```diff
@@ -383,7 +383,19 @@ export class ExperimentDetail extends React.Component {
                         {measurements && <section
                               className={classnames('measurements', { highlight: highlightMeasurementPanel })}>
                           <h3 data-l10n-id="measurements">Your privacy</h3>
-                          <div data-l10n-id={this.l10nId('measurements')} className="measurement" dangerouslySetInnerHTML={createMarkup(measurements)}></div>
+                          <div data-hook="measurements-html" className="measurement">
+                            <p data-l10n-id="experimentMeasurementIntro"
+                               data-l10n-args={JSON.stringify({ experimentTitle: experiment.title })}>
+                               href="/privacy">
```

Comment:
This looks like a typo. My eyes are telling me that the `<p>` tag from 387 above is closed on 388, so this looks like a lingering `href="/privacy">`.